test-data/local/snake_oil_no_data/snake_oil.ert

```diff
@@ -1,6 +1,6 @@
 QUEUE_SYSTEM LOCAL
+ECLBASE SNAKE_OIL_FIELD
```

Comment:
Could you add a small test that displays that if both `ECLBASE` and `JOBNAME` is present, `JOBNAME` will indeed be used.

Reply:
Done